Documentation/README.md

```diff
@@ -3,5 +3,5 @@
 ## Contents
 - [Getting Started](getting-started.md)
 - [API Reference](api-reference.md)
-- [Extension Usage Examples](ExampleUsage.md)
-- [Tutorials](Tutorials/README.md)
\ No newline at end of file
+- [Extension Usage Examples](ExtensionUsage.md)
```

Comment:
should we rename the file to lower case like rest of the pages
`extension-usage.md` ?

Reply:
nit: can we rename all sections to: `Getting started`, `API reference`, `Extension usage examples`